tests/functional/spec/household-relationship.spec.js

```diff
@@ -10,63 +10,6 @@ describe('Household relationship', function() {
 
   var schema = 'test_relationship_household.json'
 
-  it('Given I edit a relationship, when closing that row, I should be then able to edit the next unanswered relationship', function() {
```

Comment:
Fair enough, but only one of these tests relates to editing details. The other two test behaviour that should be be updated.